### ISSUE TITLE
Have weight and reps auto populate from previous workout.

### ISSUE THREAD

**Issue by gmillerdesigns** [noahjutz/Splitfit#70]
I'm sure I'm not the only one that would find it helpful if reps and weight from my workout were saved at the end of the a workout and auto populated when I next do that workout.

**Comment by noahjutz** [comment]
I agree, I've been planning to implement that. Thanks for the suggestion.

**Comment by mbnoimi** [comment]
+1
